strapi/strapi
# Fix admin user registration (in version 3)

### What does it do?

Regarding to https://github.com/strapi/strapi/pull/13307 this PR bypasses the default admin user sanitation to force the registrationToken property to be included in the response payload of the create user controller.

### Why is it needed?

Registration token is not added to the registration link when you add a new admin user in ADMINISTRATION PANEL > User.

### How to test it?

1. Create a new admin user
2. In the modal, the registration link should contain a valid registration token
3. Use this link to register the new admin

### Related issue(s)/PR(s)

https://github.com/strapi/strapi/pull/13307
